AndroidPayPalWebSdkIntegrationExample/app/src/main/res/layout/fragment_catalog.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".CatalogFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
We could remove this wrapping `ConstraintLayout` since it only has one child view.